notebooks/1_intelligent_agents_fall_2021/index.ipynb

```diff
@@ -0,0 +1,225 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
```

Comment:
First of all, lecture notes must be in **markdown** format. It means you should have a .md file as output, not a Jupyter notebook!